src/common/models/tables/ipblocktable.go

```diff
@@ -0,0 +1,25 @@
+package tables
+
+import ipapisdk "github.com/phoenixnap/go-sdk-bmc/ipapi"
+
+type IpBlock struct {
+	Id                   string  `header:"ID"`
+	Location             string  `header:"Location"`
+	CidrBlockSize        string  `header:"Cidr Block Size"`
+	Cidr                 string  `header:"Cidr"`
+	Status               string  `header:"Status"`
+	AssignedResourceId   *string `header:"Assigned Resource Id"`
```

Comment:
Since we have a cloumn called `ID`, should we stick to the same casing here in `Assigned Resource ID`?